### ISSUE TITLE
Problem with the latching process

### ISSUE THREAD

**Issue by edantoni96** [whatnick/STPM10_Arduino#1]
The data i got in the serial monitor on arduino seems doesnt change. Normaly the last two register should show the setting information off the IC according to the datasheet. But all the registers only show F000000 that means no data in the register. Do you have some example what happened in your serial monitor?

**Comment by whatnick** [comment]
Thanks for picking this up. I started this as a means to backup my exploration of this IC. I have kept the devkit for this on the shelf since the ATMEL/Microchip series was much easier to interface to. Please explore this and let us collaborate here. I can get you some logic analyser traces from this code if you need.

**Comment by edantoni96** [comment]
Yes, the logic trace should help discover any possibility of my problem.

**Comment by LASHKAR60** [comment]
Hi dear friend
I made the STPM10 circuit according to the data sheet and loaded your program for Arduino.
The SPI output was as below.
F1308303
380FFFFD
E33FFF0D
F0381B80
BC000002
1E004000
78000000
3C000000
which was almost constant but sometimes changed.
My problem in decoding according to the table is related to power, energy, voltage, etc.
I want to display the actual values on the LCD.
But unfortunately I did not succeed.
please help me
best regards
A.A.SHAMS
Shams.glg@gmail.com
